### PR TITLE
add user guidance of env plugin

### PR DESCRIPTION
**Pull request by Thor-wl**
ref: https://github.com/volcano-sh/volcano/issues/2152
Signed-off-by: Thor-wl <13164644535@163.com>

### CODE REVIEW

**Review by wpeng102 — COMMENTED** [review]
/lgtm

**Review by shinytang6 — COMMENTED** [review]
/lgtm

File: docs/user-guide/how_to_use_env_plugin.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+# Env Plugin User Guidance
```
Comment (william-wang) [inline docs/user-guide/how_to_use_env_plugin.md:1]: Change title and name to "Volcano job env plugin" to avoid causing consusion with scheduler plugin

**Review by william-wang — CHANGES_REQUESTED** [review]

**Comment by william-wang** [comment]
suggest add guide to website documention as well.

File: docs/user-guide/how_to_use_env_plugin.md (hunk 1 above)
Comment (Thor-wl) [reply docs/user-guide/how_to_use_env_plugin.md:1]: OK, let me give an update.

**Comment by Thor-wl** [comment]
> suggest add guide to website documention as well.

Got it. All the docs will be updated in the website.

**Review by william-wang — COMMENTED** [review]
/lgtm

**Review by wpeng102 — APPROVED** [review]

**Pull request merged**